### DISCUSSION TITLE
Erro ao importar arquivo mockoon

### DISCUSSION THREAD

**Discussion by matheusvarela** [balta-io/7188#26] [Perguntas e Respostas] [answered]
Boa noite
Estou tendo problema com arquivo .json que é importado para o mockoon v1.17.0

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @matheusvarela, qual problema está tendo?

**Reply by matheusvarela** [reply]
Ele chega a ler, mas não carrega as configurações
E após a leitura o arquivo é modificado perdendo a configuração inicial

**Reply by andrebaltieri** [reply]
Quais passos você executou para importar o arquivo?

**Reply by andrebaltieri** [reply]
Este link deve ajudar:
https://github.com/balta-io/7187/discussions/60#discussioncomment-2372392

**Comment by VictoriaArantes10** [comment]
Bom dia @andrebaltieri , estou tendo problemas ao abrir o arquivo ..
Quando importo o arquivo não aparece, pode me ajudar?
![1](https://user-images.githubusercontent.com/101606738/186159988-4406e4d7-3687-43a2-95aa-0d99cae54094.png)
![2](https://user-images.githubusercontent.com/101606738/186159998-63e3a3ca-3611-4dea-9305-089bbd6391fa.png)
![3](https://user-images.githubusercontent.com/101606738/186159999-ff8467b6-0635-4fda-8fc3-d4e9eefe2ed5.png)

**Reply by andrebaltieri** [reply]
@VictoriaArantes10 veja se este link ajuda => https://github.com/balta-io/7187/discussions/60#discussioncomment-2372392
